docs/python/inference/notebooks/onnxruntime-nuphar-tutorial.ipynb

```diff
@@ -2,191 +2,197 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
```

Comment:
lots of changes in this file, perhaps some were unintentional?

Reply:
reverted notebook to remove changes from this pr